src/pages/Pokemon/Stats.js

```diff
@@ -1,26 +1,58 @@
 import React from 'react';
 import './pokemon.css'
 
-
 const Stats = ({ pokemonItem }) => {
   let statsArray = pokemonItem.stats;
-    return(
-        <div className='stats-wrapper'>
-        <div className="statsDiv">
-            <h3 className='stats-header'>Stats</h3>
-        {statsArray.map((item) => {
-          return (
-            <div
-              className={`statsCard ${item.stat.name}`}
-              key={item.stat.name}>
-              {item.stat.name.charAt(0).toUpperCase() +
-                item.stat.name.slice(1)}
-            </div>
-          );
-        })}
-      </div>
+  let numsArray = []
+  for (let i = 0; i < statsArray.length; i++) {
+    const newObj = statsArray[i]
+    const nums = (newObj.base_stat)
+    numsArray.push(Math.round(nums / 17))
+  }
+
+  const StatBar = ({ numOfBarsColored, statName }) => {
+
+    const numThing = numOfBarsColored
+    const statsArray = statName
```

Comment:
mate we probs don't need to use a new const here, we can just use `statName` directly on line 26. Otherwise it's a bit confusing cos `statsArray` sounds like it should hold an array rather than just the name